viite-UI/src/model/RoadAddressProjectCollection.js

```diff
@@ -135,22 +139,29 @@
 
       projectLinksSaved = projectLinksSaved.concat(linkIds);
 
-      var projectId = currentProject.id;
+      var projectId = projectinfo.id;
 
       var data = {'linkIds': linkIds, 'projectId': projectId, 'newStatus': STATUS_TERMINATED};
 
+      console.log(data);
+      console.log(linkIds);
       if(!_.isEmpty(linkIds) && typeof projectId !== 'undefined' && projectId !== 0){
         backend.updateProjectLinks(data, function(errorObject) {
           if (errorObject.status == INTERNAL_SERVER_ERROR_500 || errorObject.status == BAD_REQUEST_400) {
             eventbus.trigger('roadAddress:projectLinksUpdateFailed', errorObject.status);
+          } else {
+            console.log("TODO Success, refresh now");
```

Comment:
The code is never entering in the else clause, because this is only a failure event. When we have success on the saving process, we are calling the roadAddress:projectLinksUpdated event, and the roads are updated there.